.editorconfig

```diff
@@ -0,0 +1,32 @@
+root = true
```

Comment:
This has been included to make sure all devs follow the same styling, editorconfig is not IDE specific

Reply:
Hmm. OK.

Reply:
I meant to say it's not specific to a single IDE. Had problems in the past where people push up pr's with all sorts of odd spacing issues or tab issues because of the editor set up. This plugin enforces styling but you have to have it installed on your editor.

I will mention that in the README

Reply:
Excellent - thank you. 

rubocop should sort out a lot of the issues you mention, but not for all the file types. So I think this does make sense.

Ideally, this deserves a PR of it's own as that would also provide a good location for any discussion about which options were set.

Reply:
Oh! and I've certainly been caught out with developers having their IDEs set to insert tabs. That's can be a real pain to debug.